src/base_object.h

```diff
@@ -40,7 +40,7 @@ class TransferData;
 
 class BaseObject : public MemoryRetainer {
  public:
-  enum InternalFields { kSlot, kInternalFieldCount };
+  enum InternalFields { kWrapperType, kSlot, kInternalFieldCount };
```

Comment:
Maybe change this to `kNodeBaseObjectType`? I think we have some other objects that aren't BaseObjects, 
 (AFAIK, `ContextifyContext`), you might want to update that as well.